languages/ruby/lib/oso/polar/polar.rb

```diff
@@ -105,7 +105,7 @@ def load_str(str, filename: nil) # rubocop:disable Metrics/MethodLength
           begin
             Query.new(next_query, host: host).results.next
           rescue StopIteration
-            raise InlineQueryFailedError
+            raise InlineQueryFailedError, format('Inline query failed: %<source>s', source: next_query.source)
```

Comment:
This formatting should be handled in the `InlineQueryFailedError#initialize` method:
```rb
    class InlineQueryFailedError < PolarRuntimeError # rubocop:disable Style/Documentation
      # @param file [String]
      def initialize(source)
        super(format('Inline query failed: %<source>s', source: source))
      end
    end
```

and then this becomes:
```suggestion
            raise InlineQueryFailedError, next_query.source
```